web/views/club_view.ex

```diff
@@ -1,3 +1,8 @@
 defmodule Gatherto.ClubView do
   use Gatherto.Web, :view
+
+  def google_maps_api do
+    api_key = System.get_env("GOOGLE_MAPS_API_KEY")
+    "https://maps.googleapis.com/maps/api/js?key=#{api_key}&libraries=places&callback=initMap"
+  end
```

Comment:
So this exact code is in `run_view.ex`, I was thinking of moving it to somewhere both views can access it. Any thoughts on where that should go? I was thinking just the views folder, but I'm not sure if that's the best place.